tools/update-openmrs-deps.mjs

```diff
@@ -0,0 +1,43 @@
+import { execSync } from 'node:child_process';
+
+try {
+  // NB for other places use '@openmrs/*@next'; here we want to ignore patient-common-lib
+  execSync(`yarn up --fixed '@openmrs/*@next' 'openmrs@next'`, {
```

Comment:
@ibacher Your comment above seems to suggest that we exclude the patient-common-lib, but I don't see any exclusion mechanism for the patient-common-lib here, considering it's also tagged to `@next`.

Reply:
That's what I get for copying and pasting. We only want that exclusion in the chart. I'll fix that up. Thanks for the catch!